python/tvm/relay/backend/contrib/ethosu/codegen.py

```diff
@@ -336,7 +379,10 @@ def relay_to_tir_func(ext_func: relay.Function) -> tvm.tir.PrimFunc:
     # this should be a single intelligent and a composite scheduler
     # that can perform scheduling based on user inputs such as
     # scratch memory size.
-    tir_mod, const_dict = lower_to_tir(mod["main"], copy_constants())
+    try:
```

Comment:
I think we should use relay.ext.ethos-u.options either to explicitly enable or explicitly disable the cascader.
If we think, if the feature is not stable enough, we could try a explicit enable.

Once it get matures, we could make that the default. WDYT ?